charts/argo-cd/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
-version: 1.0.5
-appVersion: 2.2.4
+version: 1.0.6
```

Comment:
We are jumping versions, so this should be a 1.1.0 release

Reply:
Good catch. Updated.